test/distribution/test_distribution_sampling.py

```diff
@@ -161,6 +162,36 @@ def test_multivariate_sampling(distr, params, serialize_fn) -> None:
     )
 
 
+test_cases_dirichlet = [(Dirichlet, {"alpha": mx.nd.array([0.2, 0.4, 0.9])})]
+
+
+@pytest.mark.parametrize("distr, params", test_cases_dirichlet)
+@pytest.mark.parametrize("serialize_fn", serialize_fn_list)
+def test_dirichlet_sampling(distr, params, serialize_fn) -> None:
```

Comment:
Could we have this and `test_multivariate_sampling` merged into a single test function with cases for both `MultivariateGaussian` and `Dirichlet`? I see there are some differences in how the covariance is tested, but maybe these can be unified, not sure